s3fs/core.py

```diff
@@ -375,7 +377,10 @@ async def _connect(self, refresh=False, kwargs={}):
         self._kwargs_helper = ParamKwargsHelper(self._s3)
         return self._s3
 
-    connect = sync_wrapper(_connect)
+    async def _connect(self, refresh=False, kwargs={}):
+        return await self.set_session(refresh=refresh, **kwargs)
```

Comment:
is `_connect = set_session` good enough here? Either way, a comment saying that we may deprecate this alias in the future seems appropriate.